client/my-sites/domains/domain-management/list/helpers.ts

```diff
@@ -5,6 +5,27 @@ type FilterDomainsByOwnerType = (
 	filter: 'owned-by-me' | 'owned-by-others' | undefined
 ) => Array< ResponseDomain >;
 
+type JSONValue =
+	| string
+	| number
+	| boolean
+	| null
+	| { [ x: string ]: JSONValue }
+	| Array< JSONValue >;
+
+type Site = {
+	[ key: string ]: JSONValue;
+	options: {
+		[ key: string ]: JSONValue;
+		is_domain_only: boolean;
+	};
+};
```

Comment:
There's a `SiteData` type defined in `/state/ui/selectors/site-data.ts` which I believe we can use here